climada/hazard/base.py

```diff
@@ -743,10 +789,10 @@ def strict_positive_int_accessor(array: xr.DataArray) -> np.ndarray:
                 # The accessor for the data in the Dataset
                 accessor=[
                     lambda x: to_csr_matrix(default_accessor(x)),
-                    default_accessor,
-                    strict_positive_int_accessor,
-                    lambda x: list(default_accessor(x).flat),  # list, not np.array
+                    lambda x: maybe_repeat(default_accessor(x), num_events),
                     strict_positive_int_accessor,
```

Comment:
```suggestion
                    lambda x: strict_positive_int_accessor(x),
```
for being aligned, mainly

Reply:
It's more "symmetrical", but the lambda here is completely superfluous, so I would like to leave it out 😬 You decide!